drivers/hmis/app/models/hmis/hud/processors/base.rb

```diff
@@ -290,6 +290,28 @@ def normalize_single_custom_field_value(cded, value)
       else
         raise "unexpected value \"#{value}\""
       end
+    when 'file'
```

Comment:
Just to note, saving without submitting is currently not really working. I'll start a thread to ask about this on the frontend PR.